artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/FileLockNodeManager.java

```diff
@@ -108,7 +108,16 @@ public synchronized void start() throws Exception {
    protected synchronized void setUpServerLockFile() throws IOException {
       super.setUpServerLockFile();
 
-      for (int i = 0; i < 3; i++) {
+      if (lockChannels[0] == null || !lockChannels[0].isOpen()) {
+         File fileLock = newFile(NodeManager.SERVER_LOCK_NAME);
+         if (!fileLock.exists()) {
+            fileLock.createNewFile();
+         }
+         RandomAccessFile randomFileLock = new RandomAccessFile(fileLock, "rw");
```

Comment:
Not important if we're in hurry, but you can use `FileChannel.open(file.toPath(), StandardOpenOption.CREATE,StandardOpenOption.WRITE, StandardOpenOption.READ)`

Reply:
It's not clear to me what specific benefit that would provide.

Reply:
You save to create one instance that will reference the same file descriptor, but as I've said is a minor thing :)

Reply:
we would have to change it everywhere. if you feel compelled to do it, make it a separate task?